src/EmotiBitWiFiHost.h

```diff
@@ -75,6 +85,9 @@ class EmotiBitWiFiHost
 
 	~EmotiBitWiFiHost();
 	int8_t begin();
+	void EmotiBitWiFiHost::getAvailableSubnets();
+	void getAdvertisingIpSend();
+	void getAdvertisingIpReceive();
```

Comment:
maybe change `getAdvertisingIpSend()` to `pingAvailableSubnets()` and 
change `getAdvertisingIpReceive` to `updateAdvertisingIpList()`.

I'm not 100% set on the names, but `getAdvertisingIpSend` and `getAdvertisingIpReceive` don't look right to me. thoughts ? @Malighchi